assignments/this.js

```diff
@@ -13,14 +17,57 @@
 
 // code example for Window Binding
 
+// 'use strict';
+
+// function car() {
+//   console.log(this.vroom);
+// }
+
+// car();
+// const vroom = '🏎';
+
 // Principle 2
 
 // code example for Implicit Binding
 
+// let myCar = {
+//   name: 'Herby',
+//   sound: '🏎 vrroooom',
+//   car: function() {
+//     console.log(this.sound);
+//   }
+// }
+
+// myCar.car();
+
 // Principle 3
 
 // code example for New Binding
 
+// function car() {
+//   console.log(this.sound);
+// }
+
+// let myCar = {
+//   name: 'Herby',
+//   sound: '🏎 vrroooom',
+// };
+
+// myNewWhip = car.bind(myCar);
+
+// myNewWhip();
+
 // Principle 4
 
-// code example for Explicit Binding
\ No newline at end of file
+// code example for Explicit Binding
+
+function car() {
+  console.log(this.sound);
+}
+
+let myCar = {
```

Comment:
Because all of your objects are called `myCar`, the code you've written overwrites itself - I think that's why you have the above code all commented out.

Instead of commenting out the code, you can just give each new object a new name. You could name them something like `myCarImplicit`, `myCarNew`, and `myCarExplicit` so you can tell which object is an example for which type of binding.